.circleci/config.yml

```diff
@@ -58,21 +58,22 @@ executors:
           PREFECT_SERVER__DATABASE__USERNAME: prefect
           PREFECT_SERVER__DATABASE__PASSWORD: pr343ct
 
-      - image: "postgres:11"
+      - image: "bitnami/postgresql:14.4.0-debian-11-r9"
```

Comment:
Hi! Curious why you made changes to the CircleCI config

Reply:
I've reverted the postgres version

Reply:
Thanks! You may need to revert the other changes as well.

If you merge update with `master` this should be able to pass CI.